node_api/src/repositories/userRepository.ts

```diff
@@ -0,0 +1,68 @@
+import { GraphdbService, iri, sanitise } from "@/services/graphdb.service";
+import { isArrayHasLength, isObjectHasKeys } from "@im-library/helpers/DataTypeCheckers";
+import { USER } from "@im-library/vocabulary/USER";
+
+export default class UserRepository {
+  private graph: GraphdbService;
+
+  constructor() {
+    this.graph = GraphdbService.userRepo();
+  }
+
+  public async getUserTheme(user: string): Promise<any> {
+    const qry = "SELECT ?theme WHERE { " + iri(USER.NAMESPACE + user) + " " + iri(USER.USER_THEME) + " ?theme }";
+    const rs = await this.graph.execute(qry);
```

Comment:
There's potential here for a SPARQL injection so need to make this parameterized.  Imagine the `user` variable is set to `?s ?p ?o }; DELETE ?x ?y ?z; #`, your final qry would be...
```
SELECT ?theme WHERE { ?s ?p ?o };
DELETE ?x ?y ?z
# usr:<id> ?theme
```
which would first select everything from the db, then (if that didnt bring the server down), it would delete everything.

So to do that with parameters...
```
    const qry = "SELECT ?theme WHERE { ?user ?hasTheme ?theme }";
    const rs = await this.graph.execute(qry, {
        user: iri(USER.NAMESPACE + user),
        hasTheme: iri(USER.USER_THEME)
    });
```

Obviously same goes for the remaining SPARQL below, just didnt want to add a comment for each :)